### PR TITLE
Add binary trie module part 1

### PR DESCRIPTION
**Pull request by NIC619**
### How was it fixed?
Add binary trie part 1: Add the constants and the binary encoding/decoding utilities for binary trie

#### Cute Animal Picture

Cute animal picture

### CODE REVIEW

**Comment by pipermerriam** [comment]
There are some minor linting error that CI is going to complain about.  You can find/test these locally using `tox flake8` or `flake8 trie` (you may need to install `tox` or `flake8` in your python environment).  Otherwise looks like a good start.  Please assign me for review or add the "Ready for Review" label to anything that needs to be reviewed.

**Comment by NIC619** [comment]
`flake8`'ed and reviewer assigned. Thanks!

File: trie/constants.py [hunk 1]
```diff
@@ -14,3 +14,16 @@
 NODE_TYPE_LEAF = 1
 NODE_TYPE_EXTENSION = 2
 NODE_TYPE_BRANCH = 3
+
+# Constants for Binary Trie
+TWO_BITS = [bytes([0, 0]), bytes([0, 1]),
+            bytes([1, 0]), bytes([1, 1])]
+PREFIX_00 = TWO_BITS[0]
```
Comment (pipermerriam) [inline trie/constants.py]: *minor nitpick*

I would *suggest* not making this constant reference the previous one and just declaring it independently in the spirit of decoupling

**Review by pipermerriam — CHANGES_REQUESTED** [review]
Can you get tests added for the utility functions you added.  Lets be sure they behave as expected as well as the tests serving as a spec for how they should be used.

**Comment by NIC619** [comment]
Closed in favor of #11

**Pull request closed without merging**